src/pmr/RadTransWallElemKernel.C

```diff
@@ -31,15 +31,15 @@ RadTransWallElemKernel<BcAlgTraits>::RadTransWallElemKernel(
   ElemDataRequests &dataPreReqs)
   : Kernel(),
     radEqSystem_(radEqSystem),
-    ipNodeMap_(sierra::nalu::MasterElementRepo::get_surface_master_element(BcAlgTraits::faceTopo_)->ipNodeMap())
+    ipNodeMap_(sierra::nalu::MasterElementRepo::get_surface_master_element(BcAlgTraits::topo_)->ipNodeMap())
```

Comment:
Why this change? I can not seem to see any changes in BcAlgTraits despite a number of new AlgTraits.